### DISCUSSION TITLE
light theme

### DISCUSSION THREAD

**Discussion by MAAB-FW** [Alex313031/thorium#619] [Q&A]
![image](https://github.com/Alex313031/thorium/assets/128562152/023a2e6f-c6cc-4914-94bc-002d491e0ef9)
Why is there no light theme here?

**Comment by Alex313031** [comment]
@MAAB-FW Thorium follows the system theme.

However, there is also this flag > `chrome://flags/#force-dark-mode` which if set to enabled will force it to dark mode all the time. You might also try setting it to disabled.

**Reply by MAAB-FW** [reply]
![image](https://github.com/Alex313031/thorium/assets/128562152/6cd80129-b776-4d7d-86d3-d3d3c46f09a5)
sorry it doesn't work at all

**Reply by gz83** [reply]
It seems that this problem can only be solved by changing the system's theme to a light theme.

**Reply by MAAB-FW** [reply]
so, sad. if a user want to apply dark theme in windows and light theme in browser it can't be possible for thorium...
